modules/base/mak/cortexM4/gcc/lpc43xx/Makefile

```diff
@@ -31,7 +31,7 @@
 # POSSIBILITY OF SUCH DAMAGE.
 #
 ###############################################################################
-LFLAGS += -T "externals/platforms/cortexM4/lpc43xx/linker/ciaa_lpc4337.ld"
+LFLAGS += -nostdlib -T "externals/platforms/cortexM4/lpc43xx/linker/ciaa_lpc4337.ld"
```

Comment:
La diferencia en los flags del linker entre el NXP y el de FLS es justamente ese. En NXP se utiliza el flag -nostandlib debido a que no utiliza los archivos estándard de startup del sistema (es decir para NXP en el firmware está definido todo el startup en cr_startup_lpc43xx.c  en la rutina ResetISR(void), incluso el salto hacia "main()"). Esto es distinto para la implementación estándard en FLS, ya que requiere la librería crt0.o de ARM. La explicación está clara en el siguiente link http://mcuoneclipse.com/2014/06/15/tutorial-diy-kinetis-sdk-project-with-eclipse-startup/ (en la sección Startup Code). En dicho link en la sección "Linker and Libraries" explica por qué usar el flag --specs=nosys.specs.